transaction/transactor.go

```diff
@@ -222,19 +225,26 @@ func Capture(logger kitlog.Logger) kithttp.RequestFunc {
 			source = host
 		}
 
```

Comment:
The goal of this function isn't to log anything - it's to add these values to the logger so later logs have this information.  I'm not sure why the message key is being set, I would just remove that.  The previous code added all key values pairs at the same time, but you really don't have to do that...